src/scripts.js

```diff
@@ -22,6 +22,8 @@ let searchForm = document.querySelector("#search");
 let searchInput = document.querySelector("#search-input");
 let showPantryRecipes = document.querySelector(".show-pantry-recipes-btn");
 let tagList = document.querySelector(".tag-list");
+let modifyPantryBtn = document.querySelector(".modify-pantry-btn")
+let searchIngBtn = document.querySelector(".search-ingredients-btn")
```

Comment:
Is Ing descriptive enough?

Reply:
Agreed! Let's tackle this in our big refactor near the end of the project